assets/src/edit-story/components/panels/sizePosition.js

```diff
@@ -47,10 +47,20 @@ const BoxedNumeric = styled(Numeric)`
   border-radius: 4px;
 `;
 
+const StyledToggle = styled(Toggle)`
+  + span {
```

Comment:
What does "+" do here? Is that for the sibling selector? Can we instead place margin on the right styled element instead? Siblings selectors are rather fragile and perform really poorly.